web/src/App.js

```diff
@@ -1,19 +1,23 @@
 import React from 'react';
 import Landing from './pages/landing';
-import ApolloClient from "apollo-boost";
-import { ApolloProvider } from "react-apollo";
+import ApolloClient from 'apollo-boost';
+import {ApolloProvider} from 'react-apollo';
+import NavBar from './components/navigation/navbar';
 
 const client = new ApolloClient({
-    uri: "http://localhost:4000"
+  uri: 'http://localhost:4000',
 });
 
 function App() {
+  // TODO : add a state to check if the user is logged in now
   return (
       <div>
+        <NavBar/>
+        {/* TODO : set this to sign out when the user is logged in */}
         Main
```

Comment:
Looks like this need extra indentation

Reply:
for NavBar? or TODO? I don't understand exactly.